frontend/packages/client/src/components/Community/JoinCommunityButton.js

```diff
@@ -62,7 +64,9 @@ export default function JoinCommunityButton({
     }
   };
 
-  const leaveCommunity = async () => {
+  const leaveCommunity = async (event) => {
+    event.preventDefault();
+    event.stopPropagation();
```

Comment:
added this to prevent navigation from the community card to the community when clicking on join community in the home page

Reply:
With the style changes, I agree that this is a better way than having it positioned `absolute`.